pkg/kubelet/cm/dra/plugin/registration_test.go

```diff
@@ -149,15 +149,26 @@ func TestRegistrationHandler(t *testing.T) {
 					// Set expected slice fields for the next call of this reactor.
 					// The reactor will be called next time when resourceslices object is deleted
 					// by the kubelet after plugin deregistration.
-					expectedSliceFields = fields.Set{"spec.nodeName": nodeName, "spec.driver": test.pluginName}
-
+					switch len(expectedSliceFields) {
```

Comment:
even factoring in my very limited knowledge of DRA, is not clear to me why we need `case 2` now. Every other change I made I can follow and make sense to me, but this specific addition is not clear to me. Could you please elaborate a bit?

Reply:
ok, IIUC it is about experimenting with the timing and interplay between 2 plugins, which makes sense to me and I agree it is useful to have and make sure it works.